CHANGELOG.next.md

```diff
@@ -17,6 +17,10 @@ Thanks, you're awesome :-) -->
 #### Added
 
 #### Improvements
+* Allow ECS from any directory #2019
```

Comment:
Could you move these into the "Tooling and Artifact Changes" -> Improvements section?